sdk/spring/azure-spring-cloud-autoconfigure/src/test/java/com/azure/spring/cloud/autoconfigure/context/AzureContextAutoConfigurationTest.java

```diff
@@ -3,68 +3,104 @@
+import com.azure.spring.cloud.context.core.config.AzureProperties;
+import com.microsoft.azure.AzureEnvironment;
+import com.microsoft.azure.credentials.AzureTokenCredentials;
+import com.microsoft.azure.management.Azure;
 
 public class AzureContextAutoConfigurationTest {
-    private ApplicationContextRunner contextRunner =
-        new ApplicationContextRunner().withConfiguration(AutoConfigurations.of(AzureContextAutoConfiguration.class))
+    private ApplicationContextRunner contextRunner = new ApplicationContextRunner()
+            .withConfiguration(AutoConfigurations.of(AzureContextAutoConfiguration.class))
             .withUserConfiguration(TestConfiguration.class);
 
     @Test
     public void testAzurePropertiesConfigured() {
         this.contextRunner.withPropertyValues("spring.cloud.azure.credentialFilePath=credential")
-            .withPropertyValues("spring.cloud.azure.resourceGroup=group1")
-            .withPropertyValues("spring.cloud.azure.region=westUS").run(context -> {
-                assertThat(context).hasSingleBean(AzureProperties.class);
-                assertThat(context.getBean(AzureProperties.class).getCredentialFilePath()).isEqualTo("credential");
-                assertThat(context.getBean(AzureProperties.class).getResourceGroup()).isEqualTo("group1");
-                assertThat(context.getBean(AzureProperties.class).getRegion()).isEqualTo("westUS");
-                assertThat(context.getBean(AzureProperties.class).getEnvironment()).isEqualTo(AzureEnvironment.AZURE);
-            });
+                .withPropertyValues("spring.cloud.azure.resourceGroup=group1")
+                .withPropertyValues("spring.cloud.azure.region=westUS").run(context -> {
+                    assertThat(context).hasSingleBean(AzureProperties.class);
+                    assertThat(context.getBean(AzureProperties.class).getCredentialFilePath()).isEqualTo("credential");
+                    assertThat(context.getBean(AzureProperties.class).getResourceGroup()).isEqualTo("group1");
+                    assertThat(context.getBean(AzureProperties.class).getRegion()).isEqualTo("westUS");
+                    assertThat(context.getBean(AzureProperties.class).getEnvironment())
+                            .isEqualTo(AzureEnvironment.AZURE);
+                });
     }
 
     @Test
     public void testRequiredAzureProperties() {
         this.contextRunner.withPropertyValues("spring.cloud.azure.credentialFilePath=credential")
-            .withPropertyValues("spring.cloud.azure.resourceGroup=group1").run(context -> {
-                assertThat(context).hasSingleBean(AzureProperties.class);
-                assertThat(context.getBean(AzureProperties.class).getCredentialFilePath()).isEqualTo("credential");
-                assertThat(context.getBean(AzureProperties.class).getResourceGroup()).isEqualTo("group1");
-            });
+                .withPropertyValues("spring.cloud.azure.resourceGroup=group1").run(context -> {
```

Comment:
Please do not include unnecessary change.
Maybe this file can help you: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/AzureSpringCodeStyle.xml